stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/MessageComposer.kt

```diff
@@ -137,24 +144,21 @@ public fun MessageComposer(
             }
 
             Row(
-                Modifier
-                    .fillMaxWidth(),
-                verticalAlignment = CenterVertically
+                Modifier.fillMaxWidth(),
+                verticalAlignment = Bottom
             ) {
 
                 if (shouldShowIntegrations && activeAction !is Edit) {
                     integrations()
                 } else {
                     Spacer(
-                        modifier = Modifier
```

Comment:
Can we remove this block actually - Marton and I were discussing this and we feel like it's better to have the input expand to the edge of the screen (also per the design I think). 

Maybe we can remove this or resize the spacer to have it symmetrical with the right side

Reply:
Thank you so much! :]